src/wechaty.ts

```diff
@@ -311,19 +311,19 @@ export class Wechaty extends Accessory implements Sayable {
     return super.emit(event, ...args)
   }
 
-  public on (event: 'dong',         listener: string | ((this: Wechaty, data?: string) => void))                                                    : this
-  public on (event: 'error',        listener: string | ((this: Wechaty, error: Error) => void))                                                     : this
-  public on (event: 'friendship',   listener: string | ((this: Wechaty, friendship: Friendship) => void))                                           : this
-  public on (event: 'heartbeat',    listener: string | ((this: Wechaty, data: any) => void))                                                        : this
-  public on (event: 'login' | 'logout', listener: string | ((this: Wechaty, user: ContactSelf) => void))                                            : this
-  public on (event: 'message',      listener: string | ((this: Wechaty, message: Message) => void))                                                 : this
-  public on (event: 'ready',        listener: string | ((this: Wechaty) => void))                                                                   : this
-  public on (event: 'room-invite',  listener: string | ((this: Wechaty, roomInvitation: RoomInvitation) => void))                                   : this
-  public on (event: 'room-join',    listener: string | ((this: Wechaty, room: Room, inviteeList: Contact[],  inviter: Contact) => void))            : this
-  public on (event: 'room-leave',   listener: string | ((this: Wechaty, room: Room, leaverList: Contact[], remover?: Contact) => void))             : this
-  public on (event: 'room-topic',   listener: string | ((this: Wechaty, room: Room, newTopic: string, oldTopic: string, changer: Contact) => void)) : this
-  public on (event: 'scan',         listener: string | ((this: Wechaty, qrcode: string, status: ScanStatus, data?: string) => void))                : this
-  public on (event: 'start' | 'stop', listener: string | ((this: Wechaty) => void))                                                                 : this
+  public on (event: 'dong',         listener: string | ((this: Wechaty, data?: string) => void))                                                                : this
```

Comment:
Please do not include unrelated changes.

Reply:
This is aligning the `dong` event to all the other events, since this change modified the indentation of other events, is this necessary?

Reply:
Not necessary for this PR, because we have to focus on the code design.

We can fix the align after this PR had been merged by another PR, or we add the align as the lastest commit after this PR had been approved and ready to be merged.

Reply:
okay, will revert this change.